app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala

```diff
@@ -135,6 +142,37 @@ class MovementsController @Inject() (
       case None    => updateMovementLargeMessage(movementId, triggerId)
     }
 
+  def updateMessage(movementId: MovementId, messageId: MessageId) =
+    Action.async(parse.json) {
+      implicit request =>
+        mapRequest(request.body)
```

Comment:
we can move handleUpdateMessage into this function since it's simple now.

(for {
 messageMetadata <- mapRequest(request.body)
  update <- repo.updateMessage(movementId, messageId, largeMessageMetadata, OffsetDateTime.ofInstant(clock.instant, ZoneOffset.UTC)).asPresentation
} yield update).fold[Result](
             presentationError => Status(presentationError.code.statusCode)(Json.toJson(presentationError)),
             _ => Ok
           )